SonarQube.Common/ProcessRunnerArguments.cs

```diff
@@ -10,7 +10,6 @@
 using System.Diagnostics;
 using System.Linq;
 using System.Text;
-using System.Text.RegularExpressions;
```

Comment:
I reverted my previous fix for this bug so many of these files are the results of "git revert"